presto-main/src/main/java/io/prestosql/metadata/Metadata.java

```diff
@@ -53,20 +53,14 @@
 
 public interface Metadata
```

Comment:
What about `ConnectorMetadata`?

Reply:
We should follow up on that and probably organize the connector methods in parallel with metadata, but this is just a first step.